app/models/job_application.rb

```diff
@@ -0,0 +1,13 @@
+class JobApplication < ApplicationRecord
+  belongs_to :company
+  belongs_to :user
+
+  validates :position_title, presence: true
+  validates :date_applied, presence: true
+  validates :status, presence: true
+  validates :notes, presence: true
+  validates :job_description, presence: true
+  validates :application_url, presence: true
```

Comment:
We talked about this on zoom, I am going to make a new ticket for validating  the uniqueness of job apps, this will prevent a user from saving multiple duplicate instances of job applications.